views/components/ChartList.jsx

```diff
@@ -34,50 +36,58 @@ const ChartList = ({
           </TableRow>
         </TableHead>
         <TableBody>
-          {chartList.map((chart) => (
-            <TableRow key={chart._id}>
-              <TableCell align='center'>
-                <Link color='textPrimary' href={routes.chart(chart._id)}>
-                  {chart.title?.toUpperCase()}
-                </Link>
-              </TableCell>
-              <TableCell align='center'>
-                {chart.description?.toUpperCase()}
-              </TableCell>
-              <TableCell align='center'>
-                <Button
-                  type='button'
-                  variant='text'
-                  onClick={() => onDuplicateChart(chart._id)}
-                >
-                  <PlaylistAdd />
-                </Button>
-              </TableCell>
-              <TableCell align='center'>
-                <Link href={routes.editChart(chart._id)} color='textPrimary'>
-                  <Edit />
-                </Link>
-              </TableCell>
-              <TableCell align='center'>
-                <Button
-                  type='button'
-                  variant='text'
-                  onClick={() => handleShareChart(chart)}
-                >
-                  <Share />
-                </Button>
-              </TableCell>
-              <TableCell align='center'>
-                <Button
-                  type='button'
-                  variant='text'
-                  onClick={() => removeChart(chart._id)}
-                >
-                  <Delete />
-                </Button>
-              </TableCell>
-            </TableRow>
-          ))}
+          {chartList.map((chart) => {
+            const isEditDisabled = user.uid !== chart.owner
+
+            return (
+              <TableRow key={chart._id}>
+                <TableCell align='center'>
+                  <Link color='textPrimary' href={routes.chart(chart._id)}>
+                    {chart?.title?.toUpperCase()}
+                  </Link>
+                </TableCell>
+                <TableCell align='center'>
+                  {chart?.description?.toUpperCase()}
+                </TableCell>
+                <TableCell align='center'>
+                  <Button
+                    type='button'
+                    variant='text'
+                    onClick={() => onDuplicateChart(chart._id)}
+                  >
+                    <PlaylistAdd />
+                  </Button>
+                </TableCell>
+                <TableCell align='center'>
+                  <Link
+                    href={!isEditDisabled ? routes.editChart(chart._id) : ''}
```

Comment:
Can we switch this to avoid the double-negative?

```suggestion
                    href={isEditDisabled ? '' : routes.editChart(chart._id)}
```

Reply:
https://github.com/AMP-SCZ/dpdash/pull/135/commits/0f1c63d004fcc0ce722558e1b2dab472de9388c3